File: Wordplay/python/11.py
# How many words contain the substring "TYPE”?
def main():
    in_file = open("../sowpods.txt", "r")
    type_cnt = 0
    for words in in_file:
        if("TYPE" in words):
            type_cnt += 1
    print(type_cnt)
    
if __name__ == '__main__':
    main()

File: Wordplay/js/11.js
// How many words contain the substring "TYPE”?

// Function to check words for substring
let check_contains = (word) => {

    if(word.includes("TYPE") == true) {
        return true;
    };
    return false;
};

// Main function call handles sending words to check_contains()
let main = () => {

    const fs = require("fs");
    let buffer_arry = fs.readFileSync("Wordplay/sowpods.txt", "utf-8");
    let in_file = buffer_arry.split("\n");
    let word_count = 0;

    for(let i = 0; i < in_file.length; i++) {
        if(check_contains(in_file[i])) {
            word_count += 1;
        };
    };

    console.log(word_count);
};

main();
